python_modules/dagster/dagster/components/list/list.py

```diff
@@ -168,20 +168,14 @@ def list_definitions(
                     automation_condition=node.automation_condition.get_label()
                     if node.automation_condition
                     else None,
-                    tags=sorted(list(node.tags.items())),
```

Comment:
why not update IGNORE_METADATA_KEYS_LIST_DEFINITIONS instead?

Reply:
my thinking was that the existing approach of just `str()`-ing the `MetadataValue` class and getting the `namedtuple.__str__` output was questionable so just to remove it for now and bring it back in the future with more care when we know better what exactly would be useful